.github/actions/e2e/action.yaml

```diff
@@ -77,7 +90,7 @@ runs:
     run: |
       # Build image
       echo "::group::createImage.sh"
-      ./armada-spark/scripts/createImage.sh
+      ./armada-spark/scripts/createImage.sh -p
```

Comment:
Can we build and run with and without `-p`?

Reply:
This is a good idea, but Rich is making a big change to the e2e tests right now.  If it is ok with you, I'll leave this as it is and create a new ticket to address it once his changes land.

Reply:
Have you tested this then manually with 
```
./armada-spark/scripts/createImage.sh
./armada-spark/scripts/submitSparkPi.sh
```